cdap-ui/app/cdap/components/Footer/Footer.scss

```diff
@@ -42,4 +42,11 @@ footer.app-footer {
     right: 10px;
     color: $grey-05;
   }
+
+  .selected-namespace {
```

Comment:
@akbar-sharifi Since there is no time constraint can you help remove the scss usage and define the styles as css-in-js?